src/libraries/Common/tests/TestUtilities/TestUtilities.csproj

```diff
@@ -93,6 +93,8 @@
     <ProjectReference Include="$(LibrariesProjectRoot)System.Security.Principal.Windows\ref\System.Security.Principal.Windows.csproj" PrivateAssets="all" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetFramework)' == 'net461'">
+    <!-- Upgrade the NetStandard.Library transitive xunit dependency to avoid 1.x NS dependencies. -->
+    <PackageReference Include="NETStandard.Library" Version="$(NetStandardLibraryVersion)" />
```

Comment:
We already have a packagereference for NS.Library above. If this needs to apply for all tfms, please change that one instead of adding another PackageReference.